src/components/ItemList/ItemList.js

```diff
@@ -2,18 +2,28 @@ import React from 'react';
 import Item from '../Item/Item';
 import styles from './ItemList.module.css';
 import PropTypes from 'prop-types';
+import noItems from './no-items.png';
 
-const ItemList = ({items, onClickDone, onClickDelete}) => (<ul className={styles.itemlist}>
-  {items.map(item => <li key={item.id}>
-    <Item 
-      value={item.value} 
-      isDone={item.isDone} 
-      onClickDone={onClickDone} 
-      id={item.id}
-      onClickDelete={onClickDelete}
-    />
-  </li>)}
-</ul>);
+const ItemList = ({items, moveItems}) => {
+  return (
+    <>
+      {!items.length && <div className={styles.noitems}><img src={noItems}></img></div>}
+      <ul className={styles.itemlist}>
+        {items.filter(item => item.hide == false).map((item, index) => <li key={item.id}>
```

Comment:
Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/WebHeroSchool/agatta17_todo/pullRequest?prid=6081322)